protocols/util/avro_factory.py

```diff
@@ -0,0 +1,57 @@
+import sys
+import factory.fuzzy
+from factory.base import FactoryMetaClass, BaseFactory, BaseMeta
+
+
+def mock_schema(protocol):
+    """
+    :type protocol: Protocol
+    """
+    INT_MIN_VALUE = -(1 << 31)
+    INT_MAX_VALUE = (1 << 31) - 1
+    LONG_MIN_VALUE = -(1 << 63)
+    LONG_MAX_VALUE = (1 << 63) - 1
```

Comment:
haha, @priesgo I didn't realise it was copied from exactly the same code. We should really refactor all of that soon anyway.